<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>1.1 Lightning talk | eRum2020 Program</title>
  <meta name="description" content="1.1 Lightning talk | eRum2020 Program" />
  <meta name="generator" content="bookdown 0.18 and GitBook 2.6.7" />

  <meta property="og:title" content="1.1 Lightning talk | eRum2020 Program" />
  <meta property="og:type" content="book" />
  
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="1.1 Lightning talk | eRum2020 Program" />
  
  
  

<meta name="author" content="eRum2020 organizing Committee" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="index.html"/>
<link rel="next" href="poster.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />












</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> eRum2020 Program Overview</a><ul>
<li class="chapter" data-level="1.1" data-path="lightning-talk.html"><a href="lightning-talk.html"><i class="fa fa-check"></i><b>1.1</b> Lightning talk</a><ul>
<li class="chapter" data-level="1.1.1" data-path="lightning-talk.html"><a href="lightning-talk.html#an-enriched-disease-risk-assessment-model-based-on-historical-blood-donors-records"><i class="fa fa-check"></i><b>1.1.1</b> An enriched disease risk assessment model based on historical blood donors records</a></li>
<li class="chapter" data-level="1.1.2" data-path="lightning-talk.html"><a href="lightning-talk.html#automate-flexdashboard-with-github"><i class="fa fa-check"></i><b>1.1.2</b> Automate flexdashboard with GitHub</a></li>
<li class="chapter" data-level="1.1.3" data-path="lightning-talk.html"><a href="lightning-talk.html#supporting-twitter-analytics-application-with-graph-databases-and-the-arangodb-package"><i class="fa fa-check"></i><b>1.1.3</b> Supporting Twitter analytics application with graph-databases and the aRangodb package</a></li>
<li class="chapter" data-level="1.1.4" data-path="lightning-talk.html"><a href="lightning-talk.html#using-open-access-data-to-derive-genome-composition-of-emerging-viruses"><i class="fa fa-check"></i><b>1.1.4</b> Using open-access data to derive genome composition of emerging viruses</a></li>
<li class="chapter" data-level="1.1.5" data-path="lightning-talk.html"><a href="lightning-talk.html#ptmixed-an-r-package-for-flexible-modelling-of-longitudinal-overdispersed-count-data"><i class="fa fa-check"></i><b>1.1.5</b> ptmixed: an R package for flexible modelling of longitudinal overdispersed count data</a></li>
<li class="chapter" data-level="1.1.6" data-path="lightning-talk.html"><a href="lightning-talk.html#one-way-non-normal-anova-in-reliability-analysis-using-with-doex"><i class="fa fa-check"></i><b>1.1.6</b> One-way non-normal ANOVA in reliability analysis using with doex</a></li>
<li class="chapter" data-level="1.1.7" data-path="lightning-talk.html"><a href="lightning-talk.html#towards-more-structured-data-quality-assessment-in-the-process-mining-field-the-daqapo-package"><i class="fa fa-check"></i><b>1.1.7</b> Towards more structured data quality assessment in the process mining field: the DaQAPO package</a></li>
<li class="chapter" data-level="1.1.8" data-path="lightning-talk.html"><a href="lightning-talk.html#supporting-r-in-the-binder-community"><i class="fa fa-check"></i><b>1.1.8</b> Supporting R in the Binder Community</a></li>
<li class="chapter" data-level="1.1.9" data-path="lightning-talk.html"><a href="lightning-talk.html#time-series-missing-data-visualizations"><i class="fa fa-check"></i><b>1.1.9</b> Time Series Missing Data Visualizations</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="poster.html"><a href="poster.html"><i class="fa fa-check"></i><b>1.2</b> Poster</a><ul>
<li class="chapter" data-level="1.2.1" data-path="poster.html"><a href="poster.html#modified-likelihood-ratio-model-for-handwriting-recognition-in-forensic-science."><i class="fa fa-check"></i><b>1.2.1</b> Modified likelihood ratio model for handwriting recognition in forensic science.</a></li>
<li class="chapter" data-level="1.2.2" data-path="poster.html"><a href="poster.html#guidance-for-teaching-r-to-non-programmers"><i class="fa fa-check"></i><b>1.2.2</b> Guidance for teaching R to non-programmers</a></li>
<li class="chapter" data-level="1.2.3" data-path="poster.html"><a href="poster.html#how-r-hub-can-help-you-develop-and-maintain-your-r-packages"><i class="fa fa-check"></i><b>1.2.3</b> How R-hub can help you develop and maintain your R packages</a></li>
<li class="chapter" data-level="1.2.4" data-path="poster.html"><a href="poster.html#partitional-clustering-with-extensions"><i class="fa fa-check"></i><b>1.2.4</b> Partitional clustering with extensions</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="regular-talk.html"><a href="regular-talk.html"><i class="fa fa-check"></i><b>1.3</b> Regular talk</a><ul>
<li class="chapter" data-level="1.3.1" data-path="regular-talk.html"><a href="regular-talk.html#the-r-package-flexreg-regression-mixture-models-for-bounded-responses"><i class="fa fa-check"></i><b>1.3.1</b> The R-package ‘FlexReg’: regression mixture models for bounded responses</a></li>
<li class="chapter" data-level="1.3.2" data-path="regular-talk.html"><a href="regular-talk.html#a-flexible-dashboard-for-monitoring-platform-trials"><i class="fa fa-check"></i><b>1.3.2</b> A flexible dashboard for monitoring platform trials</a></li>
<li class="chapter" data-level="1.3.3" data-path="regular-talk.html"><a href="regular-talk.html#using-xgboost-plumber-and-docker-in-production-to-power-a-new-banking-product"><i class="fa fa-check"></i><b>1.3.3</b> Using XGBoost, Plumber and Docker in production to power a new banking product</a></li>
<li class="chapter" data-level="1.3.4" data-path="regular-talk.html"><a href="regular-talk.html#astronomical-source-detection-and-background-separation-a-bayesian-nonparametric-approach"><i class="fa fa-check"></i><b>1.3.4</b> Astronomical source detection and background separation: a Bayesian nonparametric approach</a></li>
<li class="chapter" data-level="1.3.5" data-path="regular-talk.html"><a href="regular-talk.html#high-dimensional-sampling-and-volume-computation"><i class="fa fa-check"></i><b>1.3.5</b> High dimensional sampling and volume computation</a></li>
<li class="chapter" data-level="1.3.6" data-path="regular-talk.html"><a href="regular-talk.html#next-generation-supply-chain-planning-with-r-a-case-study"><i class="fa fa-check"></i><b>1.3.6</b> Next Generation Supply Chain Planning With R: A Case Study</a></li>
<li class="chapter" data-level="1.3.7" data-path="regular-talk.html"><a href="regular-talk.html#genetonic-enjoy-rna-seq-data-analysis-responsibly"><i class="fa fa-check"></i><b>1.3.7</b> GeneTonic: enjoy RNA-seq data analysis, responsibly</a></li>
<li class="chapter" data-level="1.3.8" data-path="regular-talk.html"><a href="regular-talk.html#a-simple-and-flexible-inactivitysleep-detection-r-package"><i class="fa fa-check"></i><b>1.3.8</b> A simple and flexible inactivity/sleep detection R package</a></li>
<li class="chapter" data-level="1.3.9" data-path="regular-talk.html"><a href="regular-talk.html#shazam-in-r-audio-analysis-using-the-av-package"><i class="fa fa-check"></i><b>1.3.9</b> Shazam in R? Audio analysis using the ‘av’ package</a></li>
<li class="chapter" data-level="1.3.10" data-path="regular-talk.html"><a href="regular-talk.html#deep-learning-and-time-series-approaches-for-improvement-of-vehicle-distribution-process"><i class="fa fa-check"></i><b>1.3.10</b> Deep learning and time series approaches for improvement of vehicle distribution process</a></li>
<li class="chapter" data-level="1.3.11" data-path="regular-talk.html"><a href="regular-talk.html#dm-working-with-relational-data-models-in-r"><i class="fa fa-check"></i><b>1.3.11</b> dm: working with relational data models in R</a></li>
<li class="chapter" data-level="1.3.12" data-path="regular-talk.html"><a href="regular-talk.html#powering-turing-e-atlas-with-r"><i class="fa fa-check"></i><b>1.3.12</b> Powering Turing e-Atlas with R</a></li>
<li class="chapter" data-level="1.3.13" data-path="regular-talk.html"><a href="regular-talk.html#an-innovative-way-to-support-your-sales-force"><i class="fa fa-check"></i><b>1.3.13</b> An innovative way to support your sales force</a></li>
<li class="chapter" data-level="1.3.14" data-path="regular-talk.html"><a href="regular-talk.html#damirseq-2.0-from-high-dimensional-data-to-cost-effective-reliable-prediction-models"><i class="fa fa-check"></i><b>1.3.14</b> DaMiRseq 2.0: from high dimensional data to cost-effective reliable prediction models</a></li>
<li class="chapter" data-level="1.3.15" data-path="regular-talk.html"><a href="regular-talk.html#interpretable-and-accessible-deep-learning-for-omics-data-with-r-and-friends"><i class="fa fa-check"></i><b>1.3.15</b> Interpretable and accessible Deep Learning for omics data with R and friends</a></li>
<li class="chapter" data-level="1.3.16" data-path="regular-talk.html"><a href="regular-talk.html#analyzing-preference-data-with-the-bayesmallows-package"><i class="fa fa-check"></i><b>1.3.16</b> Analyzing Preference Data with the BayesMallows Package</a></li>
<li class="chapter" data-level="1.3.17" data-path="regular-talk.html"><a href="regular-talk.html#analyzing-preference-data-with-the-bayesmallows-package-1"><i class="fa fa-check"></i><b>1.3.17</b> Analyzing Preference Data with the BayesMallows Package</a></li>
<li class="chapter" data-level="1.3.18" data-path="regular-talk.html"><a href="regular-talk.html#flexible-meta-analysis-of-generalized-additive-models-with-metagam"><i class="fa fa-check"></i><b>1.3.18</b> Flexible Meta-Analysis of Generalized Additive Models with metagam</a></li>
<li class="chapter" data-level="1.3.19" data-path="regular-talk.html"><a href="regular-talk.html#flexible-meta-analysis-of-generalized-additive-models-with-metagam-1"><i class="fa fa-check"></i><b>1.3.19</b> Flexible Meta-Analysis of Generalized Additive Models with metagam</a></li>
<li class="chapter" data-level="1.3.20" data-path="regular-talk.html"><a href="regular-talk.html#controlled-r-development-with-docker"><i class="fa fa-check"></i><b>1.3.20</b> Controlled R development with Docker</a></li>
<li class="chapter" data-level="1.3.21" data-path="regular-talk.html"><a href="regular-talk.html#interactive-visualization-of-complex-texts"><i class="fa fa-check"></i><b>1.3.21</b> Interactive visualization of complex texts</a></li>
<li class="chapter" data-level="1.3.22" data-path="regular-talk.html"><a href="regular-talk.html#connector-a-computational-approach-to-study-intratumor-heterogeneity."><i class="fa fa-check"></i><b>1.3.22</b> CONNECTOR: a computational approach to study intratumor heterogeneity.</a></li>
<li class="chapter" data-level="1.3.23" data-path="regular-talk.html"><a href="regular-talk.html#flexible-deep-learning-via-the-juliaconnector"><i class="fa fa-check"></i><b>1.3.23</b> Flexible deep learning via the JuliaConnectoR</a></li>
<li class="chapter" data-level="1.3.24" data-path="regular-talk.html"><a href="regular-talk.html#gwqs-an-r-package-for-linear-and-generalized-weighted-quantile-sum-wqs-regression"><i class="fa fa-check"></i><b>1.3.24</b> gWQS: An R Package for Linear and Generalized Weighted Quantile Sum (WQS) Regression</a></li>
<li class="chapter" data-level="1.3.25" data-path="regular-talk.html"><a href="regular-talk.html#rlinkedcharts-a-novel-approach-for-simple-but-powerful-interactive-data-analysis"><i class="fa fa-check"></i><b>1.3.25</b> R/LinkedCharts: A novel approach for simple but powerful interactive data analysis</a></li>
<li class="chapter" data-level="1.3.26" data-path="regular-talk.html"><a href="regular-talk.html#transparent-journalism-through-the-power-of-r"><i class="fa fa-check"></i><b>1.3.26</b> Transparent Journalism Through the Power of R</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="shiny-demo.html"><a href="shiny-demo.html"><i class="fa fa-check"></i><b>1.4</b> Shiny demo</a><ul>
<li class="chapter" data-level="1.4.1" data-path="shiny-demo.html"><a href="shiny-demo.html#decision-support-for-maritime-spatial-planning"><i class="fa fa-check"></i><b>1.4.1</b> Decision support for maritime spatial planning</a></li>
<li class="chapter" data-level="1.4.2" data-path="shiny-demo.html"><a href="shiny-demo.html#automated-receptive-and-interactive-a-classroom-based-data-generation-exercise-using-shiny"><i class="fa fa-check"></i><b>1.4.2</b> Automated, receptive and interactive: a classroom-based data generation exercise using Shiny</a></li>
<li class="chapter" data-level="1.4.3" data-path="shiny-demo.html"><a href="shiny-demo.html#mobility-scan"><i class="fa fa-check"></i><b>1.4.3</b> Mobility scan</a></li>
<li class="chapter" data-level="1.4.4" data-path="shiny-demo.html"><a href="shiny-demo.html#scoring-the-implicit-association-test-has-never-been-easier-dscoreapp"><i class="fa fa-check"></i><b>1.4.4</b> Scoring the Implicit Association Test has never been easier: DscoreApp</a></li>
<li class="chapter" data-level="1.4.5" data-path="shiny-demo.html"><a href="shiny-demo.html#rtrhexng-hexagon-sticker-app-for-rtrng"><i class="fa fa-check"></i><b>1.4.5</b> rTRhexNG: Hexagon sticker app for rTRNG</a></li>
</ul></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">eRum2020 Program</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="lightning-talk" class="section level2">
<h2><span class="header-section-number">1.1</span> Lightning talk</h2>
<div id="an-enriched-disease-risk-assessment-model-based-on-historical-blood-donors-records" class="section level3">
<h3><span class="header-section-number">1.1.1</span> An enriched disease risk assessment model based on historical blood donors records</h3>
<p><em>Andrea Cappozzo, PhD student at University of Milan-Bicocca</em></p>
<p><strong>Track(s):</strong> R Applications</p>
<p><strong>Abstract:</strong></p>
<p>Historically, the medical literature has largely focused on determining risk factors at an illness-specific level. Nevertheless, recent studies suggested that identical risk factors may cause the appearance of different diseases in different patients (Meijers &amp; De Boer, 2019).</p>
<p>Thanks to the joint collaboration of Heartindata, a group of data scientists offering their passion and skills for social good, and Avis Milano, the Italian blood donor organization, an enriched disease risk assessment model is developed. Multiple risk factors and donations drop-out causes are collectively analyzed from AVIS longitudinal records, with the final aim of providing a broader and clearer overview of the interplay between risk factors and associated diseases in the blood donors population.</p>
</div>
<div id="automate-flexdashboard-with-github" class="section level3">
<h3><span class="header-section-number">1.1.2</span> Automate flexdashboard with GitHub</h3>
<p><em>Binod Jung Bogati, Data Analyst Intern at VIN</em></p>
<p><strong>Track(s):</strong> R Dataviz &amp; Shiny</p>
<p><strong>Abstract:</strong></p>
<p>flexdashboard is a great tool for building an interactive dashboard in R. We can host it for free on GitHub Pages, Rpubs and many other places.</p>
<p>Hosted flexdashboard is static so changes in our data we need to manually update and publish every time. If we want to auto-update we may need to integrate Shiny. However, it may not be suitable for every case.</p>
<p>To overcome this, we have a solution called GitHub Action. It’s a feature from GitHub which automates our tasks in a convenient way.</p>
<p>With the help of GitHub Actions, we can automate our flexdashboard (Rmarkdown) updates. It builds a container that runs our R scripts. We can trigger it every time we push on GitHub or schedule it every X minutes/hours/days/month.</p>
<p>If you want to learn more about the GitHub Action. And also know how to automate updates on your flexdashboard. Please do come and join me.</p>
</div>
<div id="supporting-twitter-analytics-application-with-graph-databases-and-the-arangodb-package" class="section level3">
<h3><span class="header-section-number">1.1.3</span> Supporting Twitter analytics application with graph-databases and the aRangodb package</h3>
<p><em>Gabriele Galatolo, Kode Srl, Software Developer &amp; Data Scientist</em></p>
<p><strong>Track(s):</strong> R Applications</p>
<p><strong>Abstract:</strong></p>
<p>The importance of finding efficient ways to model and to store unstructured data has incredibly grown in the last decade, in particular with the strong expansion of social-media services. Among those storing tools an increasingly important class of databases is represented by the graph-oriented databases, where relationships between data are considered first-class citizens.
In order to support the analyst or the data scientist to interact and use in a simple way with this paradigm, we developed last year the package aRangodb, an interface with the graph-oriented database ArangoDB.
To show the capabilities of the package and of the underlying way to model data using graphs we present Tweetmood, a tool to analyze and visualize tweets from Twitter.
In this talk, we will present some of the most significant features of the package applied in the Tweetmood context, such as functionalities to traverse the graph and some examples in which the user can elaborate those graphs to get new information that can easily be stored using the functions and the tools available in the package.</p>
</div>
<div id="using-open-access-data-to-derive-genome-composition-of-emerging-viruses" class="section level3">
<h3><span class="header-section-number">1.1.4</span> Using open-access data to derive genome composition of emerging viruses</h3>
<p><em>Liam Brierley, MRC Skills Development Fellow, University of Liverpool</em></p>
<p><strong>Track(s):</strong> R Life Sciences</p>
<p><strong>Abstract:</strong></p>
<p>Outbreaks of new viruses continue to threaten global health, including pandemic influenza, Ebola virus, and the novel coronavirus ‘nCoV-2019’. Advances in genome sequencing allow access to virus RNA sequences on an unprecedented scale, representing a powerful tool for epidemiologists to understand new viral outbreaks.</p>
<p>We use NCBI’s GenBank, a curated open-access repository containing &gt;200 million genetic sequences (3 million viral sequences) directly submitted by users, representing many individual studies. However, the resulting breadth of data and inconsistencies in metadata present consistent challenges.</p>
<p>We demonstrate our approach using R to address these challenges and a need for reproducibility as data increases. Firstly, we use <code>rentrez</code> to programmatically search, filter, and obtain virus sequences from GenBank. Secondly, we use <code>taxize</code> to resolve pervasive problems of naming conflicts, as virus names are often recorded differently between entries, partly because virus classification is complex and regularly revised. We successfully resolve 428 mammal and bird RNA viruses to species level before extracting sequences.</p>
<p>Obtaining genome sequences of a large inventory of viruses allows us to estimate genomic composition biases, which show promise in predicting virus epidemiology. Ultimately, this pathway will allow better quantification of future epidemic threats.</p>
</div>
<div id="ptmixed-an-r-package-for-flexible-modelling-of-longitudinal-overdispersed-count-data" class="section level3">
<h3><span class="header-section-number">1.1.5</span> ptmixed: an R package for flexible modelling of longitudinal overdispersed count data</h3>
<p><em>Mirko Signorelli, Dept. of Biomedical Data Sciences, Leiden University Medical Center</em></p>
<p><strong>Track(s):</strong> R Machine Learning &amp; Models, R Life Sciences</p>
<p><strong>Abstract:</strong></p>
<p>Overdispersion is a commonly encountered feature of count data, and it is usually modelled using the negative binomial (NB) distribution. However, not all overdispersed distributions are created equal: while some are severely zero-inflated, other exhibit heavy tails.
Mounting evidence from many research fields suggests that often NB models cannot fit sufficiently well heavy-tailed or zero-inflated counts. It has been proposed to solve this problem by using the more flexible Poisson-Tweedie (PT) family of distributions, of which the NB is special case. However, current methods based on the PT can only handle cross-sectional datasets and no extension for correlated data is available.
To overcome this limitation we propose a PT mixed-effects model that can be used to flexibly model longitudinal overdispersed counts. To estimate this model we develop a computational pipeline that uses adaptive quadratures to accurately approximate the likelihood of the model, and numeric optimization methods to maximize it. We have implemented this approach in the R package ptmixed, which is published on CRAN.
Besides showcasing the package’s functionalities, we will present an assessment of the accuracy of our estimation procedure, and provide an example application where we analyse longitudinal RNA-seq data, which often exhibit high levels of zero-inflation and heavy tails.</p>
</div>
<div id="one-way-non-normal-anova-in-reliability-analysis-using-with-doex" class="section level3">
<h3><span class="header-section-number">1.1.6</span> One-way non-normal ANOVA in reliability analysis using with doex</h3>
<p><em>Mustafa CAVUS, PhD Student <span class="citation">@Eskisehir</span> Technical University</em></p>
<p><strong>Track(s):</strong> R Production, R Life Sciences, R Applications</p>
<p><strong>Abstract:</strong></p>
<p>One-way ANOVA is used for testing equality of several population means in statistics, and current packages in R provides functions to apply it. However, the violation of its assumptions are normality and variance heterogeneity limits its use, also not possible in some cases. doex provides alternative statistical methods to solve this problem. It has several tests based on generalized p-value, parametric bootstrap and fiducial approaches for the violation of variance heterogeneity and normality. Moreover, it provides the newly proposed methods for testing equality of mean lifetimes under different failure rates.</p>
<p>This talk introduces doex package provides has several methods for testing equality of population means independently the strict assumptions of ANOVA. An illustrative example is given for testing equality of mean of product lifetimes under different failure rates.</p>
</div>
<div id="towards-more-structured-data-quality-assessment-in-the-process-mining-field-the-daqapo-package" class="section level3">
<h3><span class="header-section-number">1.1.7</span> Towards more structured data quality assessment in the process mining field: the DaQAPO package</h3>
<p><em>Niels Martin, Postdoctoral researcher Research Foundation Flanders (FWO) - Hasselt University</em></p>
<p><strong>Track(s):</strong> R Applications</p>
<p><strong>Abstract:</strong></p>
<p>Process mining is a research field focusing on the extraction of insights on business processes from process execution data embedded in files called event logs. Event logs are a specific data structure originating from information systems supporting a business process such as an Enterprise Resource Planning System or a Hospital Information System. As a research field, process mining predominantly focused on the development of algorithms to retrieve process insights from an event log. However, consistent with the “garbage in - garbage out”-principle, the reliability of the algorithm’s outcomes strongly depends upon the data quality of the event log. It has been widely recognized that real-life event logs typically suffer from a multitude of data quality issues, stressing the need for thorough data quality assessment. Currently, event log quality is often judged on an ad-hoc basis, entailing the risk that important issues are overlooked. Hence, the need for a more structured data quality assessment approach within the process mining field. Therefore, the DaQAPO package has been developed, which is an acronym for Data Quality Assessment of Process-Oriented data. It offers an extensive set of functions to automatically identify common data quality problems in process execution data. In this way, it is the first R-package which supports systematic data quality assessment for event data.</p>
</div>
<div id="supporting-r-in-the-binder-community" class="section level3">
<h3><span class="header-section-number">1.1.8</span> Supporting R in the Binder Community</h3>
<p><em>Sarah Gibson, Research Software Engineer and 2020 Software Sustainability Institute Fellow</em></p>
<p><strong>Track(s):</strong> R World</p>
<p><strong>Abstract:</strong></p>
<p>Binder is a project developed for the community by the community. It supports the sharing of reproducible computational analyses over the cloud in a variety of programming languages. The users of mybinder.org (the free and public Binder service) are in a privileged position to suggest the features they want to see, and even contribute them to the project. That being said, the maintainers and operators of mybinder.org are primarily skilled in Python. I would like to use my lightning talk session to begin a conversation with the R community on how Binder can better support the needs of R language users, and how they can become involved in the maintenance and development of such a project.</p>
</div>
<div id="time-series-missing-data-visualizations" class="section level3">
<h3><span class="header-section-number">1.1.9</span> Time Series Missing Data Visualizations</h3>
<p><em>Steffen Moritz, Institute for Data Science, Engineering, and Analytics, TH Köln</em></p>
<p><strong>Track(s):</strong> R Dataviz &amp; Shiny, R Applications</p>
<p><strong>Abstract:</strong></p>
<p>Missing data is a quite common problem for time series, which usually also complicates later analysis steps.
In order to deal with this problem, visualizing the missing data is a very good start.</p>
<p>Visualizing the patterns in the missing data can provide more information about the reasons for the missing data and give hints on how to best proceed with the analysis.</p>
<p>This talk gives a short intro into the new plotting functions being introduced with the 3.1 version of the imputeTS CRAN package.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="index.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="poster.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
